src/main.rs

```diff
@@ -279,9 +279,9 @@ enum Commands {
         #[arg(long, short)]
         config: Option<String>,
 
-        /// Whether to broadly profile the entire query, rather than each pattern within the query.
+        /// Whether to profile the entire query file, rather than each pattern within the query.
         #[arg(long, short)]
-        broad: bool,
+        per_file: bool,
```

Comment:
`per-file`? (Mixing dashes and underscores is always iffy -- unless Rust translates that automatically?)